dipy/reconst/dki.py

```diff
@@ -466,7 +466,8 @@ def directional_diffusion(dt, V, *, min_diffusivity=0):
     return adc
 
 
-def directional_diffusion_variance(kt, V):
+@warning_for_keywords()
+def directional_diffusion_variance(kt, V, *, min_kurtosis=-3 / 7):
```

Comment:
problem from the rebase, to put it back to `def directional_diffusion_variance(kt, V):`

Reply:
Done in 1e5c01ff2